domains/akzana.json

```diff
@@ -0,0 +1,12 @@
+{
+  "description": "Portfolio Site for Akzana. I want to host small projects on Akzana.Is-A.Dev to showcase my software engineering skills.",
+  "repo": "https://github.com/akzana/akzana.github.io",
+  "owner": {
+    "username": "akzana",
+    "email": "akzanaklepsch@outlook.com"
+  },
+  "record": {
+    "CNAME": "akzana.github.io",
+    "TXT": ["a9e7df1f4744431d00299a1ca92510"]
```

Comment:
1. You can't use CNAME with other records
2. 2. is that TXT a github pages verification string? if so you have to put it ina  different file, see https://is-a.dev/docs/github-pages-verification

Reply:
Thank you for the information. I will update this PR with the Github verification

Reply:
Hello again, 
I've updated my request with another JSON file with the github pages verification. Let me know if there's still wrong info.